lib/local-links-manager/import/interactions_importer.rb

```diff
@@ -15,16 +15,15 @@ def self.import
 
       def initialize(csv_downloader = CsvDownloader.new(CSV_URL, header_conversions: FIELD_NAME_CONVERSIONS))
         @csv_downloader = csv_downloader
-        @lgil_codes_in_csv = Set.new
+        @comparer = ImportComparer.new("interactions")
```

Comment:
For the `CsvDownloader` we used dependency injection, could do the same with `ImportComparer`?